crtauth/client.py

```diff
@@ -19,6 +19,7 @@
 import io
 import msgpack
 from crtauth import ssh, protocol, msgpack_protocol, exceptions
+from crtauth.constant_time_compare import to_i
```

Comment:
It seems conceptually wrong for such a generic function to live in `constant_time_compare.py`. Let's rename the file `util.py` or somesuch.